token/core/zkatdlog/crypto/audit/auditor.go

```diff
@@ -413,3 +409,11 @@ func GetAuditInfoForTransfers(transfers [][]byte, metadata []driver.TransferMeta
 	}
 	return auditableInputs, outputs, nil
 }
+
+func commit(vector []*math.Zr, generators []*math.G1, c *math.Curve) *math.G1 {
+	com := c.NewG1()
```

Comment:
In Gnark this produces the point to infinity, but are we sure this is true for all mathlib implementations?

Reply:
I will add a test to check that `c.NewG1()` always returns a `point_at_infinity`.